Main.java

```diff
@@ -0,0 +1,24 @@
+import exceptions.DuplicateProductException;
+import exceptions.ProductNotFoundException;
+import model.Product;
+import service.ProductManager;
+
+public class Main {
+    public static void main(String[] args) {
+        ProductManager productManager = new ProductManager();
+
+        try {
+            productManager.addProduct(new Product(1, "Laptop", 999.99, 10));
+            productManager.addProduct(new Product(2, "Phone", 599.99, 20));
+            productManager.addProduct(new Product(3, "Tablet", 299.99, 15));
+
+            productManager.updateProduct(new Product(2, "Phone", 649.99, 20));
+
+            productManager.deleteProduct(3);
+
+            productManager.sellProduct(1, 5);
+        } catch (DuplicateProductException | ProductNotFoundException e) {
```

Comment:
there should also be a generic catch for any other type of exceptions that may occur.

Reply:
Done